src/lib.rs

```diff
@@ -81,9 +81,10 @@ pub mod client;
 #[cfg(feature = "server")]
 pub mod server;
 
+#[cfg(feature = "renet_netcode")]
+pub use bevy_renet::netcode;
+
```

Comment:
Let's remove a newline to group them:
```suggestion
#[cfg(feature = "renet_netcode")]
pub use bevy_renet::netcode;
```